docs/pages/machine-id/reference/diagnostics-service.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Diagnostics Service
+description: Reference information for the `tbot` diagnostics service.
+---
+
+The `tbot` process can optionally expose a diagnostics service. This is
+disabled by default, but once enabled, allows useful information about the
+running `tbot` process to be queried via HTTP.
+
+## Configuration
+
+To enable the diagnostics service, you must specify an address and port for
+it to listen on.
+
+For security reasons, you should ensure that access to this listener is
+restricted. In most cases, the most secure thing to do is to bind the listener
+to `127.0.0.1`, which will only allow access from the local machine.
+
+You can configure the diagnostics service using the `--diag-addr` CLI parameter:
+
+```code
+$ tbot start -c my-config.yaml --diag-addr 127.0.0.1:1234
+```
+
+Or directly within the configuration file using `diag_addr`:
+
+```yaml
+diag_addr: 127.0.0.1:1234
```

Comment:
typically we give `3xxx` ports. The diagnostic one for `teleport` is `3000`.  Perhaps `3001`?

Reply:
Sounds reasonable - it's really up to the user here but makes sense to demonstrate a default that aligns with our default ports.

Reply:
4248ee6c34